source/raylib/package.d

```diff
@@ -179,7 +179,37 @@ struct Color
     ubyte r; // Color red value
     ubyte g; // Color green value
     ubyte b; // Color blue value
-    ubyte a; // Color alpha value
+    ubyte a = 255; // Color alpha value
+
+    @safe @nogc nothrow:
+
+    this(ubyte r, ubyte g, ubyte b, ubyte a) {
+        this.r = r;
+        this.g = g;
+        this.b = b;
+        this.a = a;
+    }
+
+    this(ubyte r, ubyte g, ubyte b) {
+        this(r, g, b, 255);
+    }
+
+    this(ubyte[4] rgba) {
+        this(rgba[0], rgba[1], rgba[2], rgba[3]);
+    }
+
+    this(ubyte[3] rgb) {
+        this(rgb[0], rgb[1], rgb[2], 255);
+    }
+
+    this(uint rgba) {
+        this(
+            (rgba & 0xFF000000) >> 24,
+            (rgba & 0xFF0000) >> 16,
+            (rgba & 0xFF00) >> 8,
+            (rgba & 0xFF),
+        );
+    }
```

Comment:
I get this, but I think this is presuming a format for all `uint` that are passed. This is going cause some odd cases, people might assume something different, or they might accidentally have a typo and pass e.g. `255` in here, and it compiles and does something very odd.

In addition, raylib already has the [`GetColor`](https://github.com/raysan5/raylib/blob/d9c5066382615644137b4f65479c65c44820027a/src/rtextures.c#L4805) function, which does something similar.

I'd rather see factory functions that do this if you have a specific format you want to parse (not in this PR).

Reply:
Yeah, it can create some issues.